src/Tizen.NUI/src/public/BaseComponents/Style/ViewStyleBindableProperty.cs

```diff
@@ -49,6 +49,13 @@ public partial class ViewStyle
             defaultValueCreator: (bindable) => ((ViewStyle)bindable).focusable
         );
 
+        /// <summary> Bindable property of FocusableInTouchMode. Please do not open it. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty FocusableInTouchModeProperty = BindableProperty.Create(nameof(FocusableInTouchMode), typeof(bool?), typeof(ViewStyle), null,
+            propertyChanged: (bindable, oldValue, newValue) => ((ViewStyle)bindable).focusableInTouchMode = (bool?)newValue,
+            defaultValueCreator: (bindable) => ((ViewStyle)bindable).focusableInTouchMode
```

Comment:
I think, default value should be null, because it is nullable value, remove `defaultValueCreator`